sdk-base/src/main/java/com/mapbox/maps/plugin/annotation/Annotation.kt

```diff
@@ -87,5 +92,7 @@ abstract class Annotation<T : Geometry>(
      * Minimum latitude value in Mercator projection.
      */
     const val MIN_MERCATOR_LATITUDE = -85.05112877980659
+
+    private const val FEATURE_IDENTIFIER_PREFIX = "mapbox_annotation_"
```

Comment:
nit: we can add documentation here too, for consistency.

Reply:
imho naming is self-descriptive and it's private 😉